src/mcts/params.cc

```diff
@@ -92,6 +102,8 @@ void SearchParams::Populate(OptionsParser* options) {
   options->Add<IntOption>(kMiniBatchSizeId, 1, 1024) = 1;
   options->Add<IntOption>(kMaxPrefetchBatchId, 0, 1024) = 32;
   options->Add<FloatOption>(kCpuctId, 0.0f, 100.0f) = 1.2f;
+  options->Add<FloatOption>(kContemptId, -1.0f, 1.0f) = 0.005f;
+  options->Add<FloatOption>(kContempt2Id, -1.0f, 1.0f) = 0.015f;
```

Comment:
The default for both options should be zero, and then changed in engine.cc (as is done already). This will make it safe for training.